pkg/processor/test/suite/suite.go

```diff
@@ -53,13 +54,17 @@ type OnAfterContainerRun func(deployResult *platform.CreateFunctionResult) bool
 // function container (through an trigger of some sort)
 type TestSuite struct {
 	suite.Suite
-	Logger       logger.Logger
-	DockerClient dockerclient.Client
-	Platform     platform.Platform
-	TestID       string
-	Runtime      string
-	RuntimeDir   string
-	FunctionDir  string
+	Logger                logger.Logger
+	DockerClient          dockerclient.Client
+	Platform              platform.Platform
+	TestID                string
+	Runtime               string
+	RuntimeDir            string
+	FunctionDir           string
+	PlatformType          string
```

Comment:
A bit bloated, are all these needed? (see other related comments - perhaps for future PR if not obvious fix)